test/e2e/node/pod_resize.go

```diff
@@ -1633,6 +1633,30 @@ func doPodResizeSchedulerTests() {
 	})
 }
 
+func revertResourceResizePatch(pod *v1.Pod, patch string) {
+	// Logic to revert the resource resize patch goes here
+	fmt.Println("Reverting resource resize with patch:", patch)
+
+	// Update the pod's spec with the original resource size
+	originalPod := makeTestPod(pod.Namespace, pod.Name, pod.ObjectMeta.Labels["time"], getOriginalContainerInfo(pod))
+	pod.Spec = originalPod.Spec
+
+	// Apply the updated pod spec to revert the resize
+	applyPodSpec(pod)
+}
+
+func getOriginalContainerInfo(pod *v1.Pod) []TestContainerInfo {
+	// Logic to retrieve the original container info goes here
+	// You need to implement this function based on your specific scenario
+	// Return the original container info as an array of TestContainerInfo
+	return []TestContainerInfo{}
+}
```

Comment:
Where these functions are supposed to be called?

Reply:
Hi @bart0sh, did I do something wrong? Please guide me if possible. I wrote this code to revert resize patch.

Reply:
how is this code supposed to run if is not called from any place?

Reply:
> how is this code supposed to run if is not called from any place?

Okey I'll work on it, thanks for your help!

Reply:
take your time, learn how to run these tests locally too, verify that they run, contributing is a long and slow process but you'll learn a lot on the path